config/environments/production.rb

```diff
@@ -39,7 +39,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :amazon
+  # config.active_storage.service = :local
```

Comment:
did you mean to remove amazon from production active_storage, @Brantron ?

Reply:
no, I can undo that part, it was part of my testing when deploys were failing on active storage

Reply:
it's ok. i'll go back through it. aws keys should've been in the encrypted file, but prob good to revisit that whole segment. thanks!